locale/fi/LC_MESSAGES/django.po

```diff
@@ -3529,7 +3529,11 @@ msgstr "Päävalikko"
 msgid "Skip main navigation"
 msgstr "Ohita päävalikko"
 
-#: templates/base.html:83
+#: templates/base.html:80
+msgid "Main"
+msgstr "Pää"
```

Comment:
I think these are bit too short strings to be clear. Specially the Finnish one is confusing. As such, it just means "head" as body part. To mean "main" it usually requires a context. I'm not sure if the location of the aria-label gives enough context, but it's possible.

Reply:
I'd be inclined to disagree. 

These are only used by assistive technology, and the technology that I've tested always present them in the format `[NAME] [ROLE]`. I didn't test this enough when I implemented these names to start with, so hadn't noticed that we were actually getting things like this before:

![Screenshot of the VoiceOver Rotor showing "Main navigation navigation" and some more landmarks with odd names.](https://user-images.githubusercontent.com/6674046/86582102-d0a32900-bf89-11ea-8166-e0a7c640ffc3.png)

So dropping the "navigation" part and leaving only the name seems a little more correct and understandable.

![Screenshot of the VoiceOver Rotor showing "Main navigation" and some more landmarks with correct names.](https://user-images.githubusercontent.com/6674046/86582237-00523100-bf8a-11ea-9b47-0c95708e5962.png)

Also, I'd planned on adding an explicit note about this, but the PR is still marked as a draft, so I wasn't expecting any reviews yet.

Reply:
And I double-checked how this happens in Finnish...

![Näyttökuva 2020-7-6 kello 13 29 13](https://user-images.githubusercontent.com/6674046/86583907-b0289e00-bf8c-11ea-988f-4b517373f196.png)

And it becomes:

![Näyttökuva 2020-7-6 kello 13 31 32](https://user-images.githubusercontent.com/6674046/86584057-0269bf00-bf8d-11ea-9756-6f95cd5423fa.png)

Reply:
Those spaces in Finnish cause me to cringe really, really badly. It looks like someone is horrid at spelling, since those could very well be compound words mostly, but grammatically could not be written like that.

Reply:
I'd assume that things like this are familiar enough to Finnish-speaking regular users of assistive technology though. The second word is the role, so this is the expected behaviour. 

Or do you think we should actually go with `Päänavigointi navigointi` here?

Reply:
(The second screenshot ("pää navigointi" and "kurssi navigointi") is actually giving me serious anxiety ("pää navigointi" and "kurssi navigointi"). The first Finnish version doesn't seem nearly as bad somehow, I guess it's easier to consider the things as separate entities rather than butchered spellings. Of course, if this would never be seen, just read aloud by something such as a screen reader, the second version isn't bad.)

Reply:
Here's vero.fi in Finnish

<img width="436" alt="Näyttökuva 2020-7-6 kello 14 18 35" src="https://user-images.githubusercontent.com/6674046/86588098-b0786780-bf93-11ea-83e3-ec0277d1188d.png">

If you think it's much better, I don't mind going with something like this, but it does feel as though it contradicts the W3C guidelines:

> Avoid using the landmark role as part of the label. For example, a navigation landmark with a label "Site Navigation" will be announced by a screen reader as "Site Navigation Navigation". The label should simply be "Site".

What do you think @etanttila ?

Reply:
I really don't know. I have so little experience with that kind of stuff (I don't even know how to view landmarks on a site). Why can't there be specific guidelines for this in Finnish (and possibly other languages that have compound words)? :/ It would be interesting to hear the thoughts of a person or people who are Finnish speaking and are used to using these landmarks. My gut feeling is that the latter one is quite a bit better (especially in written form), but I assume that what people are actually used to affects which is better.

Reply:
I'll resolve this conversation now. There's potentially more to discuss, but so many things have changed, let's start the conversation again at the appropriate time.